src/runtime_src/core/tools/xbmgmt2/OO_Hotplug.cpp

```diff
@@ -68,7 +66,7 @@ OO_Hotplug::execute(const SubCmdOptions& _options) const
 
   XBUtilities::verbose("Option(s):");
   for (const auto & aString : _options)
-    XBUtilities::verbose(std::string(" ") + aString);
+    XBUtilities::verbose(" " + aString);
```

Comment:
Hmm...
I don't think this works the way you think it does for how does the compiler know to concatenate a C style string with a std::string object.